{{ cookiecutter.package_name }}/.gitignore

```
@@ -55,6 +55,7 @@ distribute-*.tar.gz
 .project
 .pydevproject
 .settings
+pip-wheel-metadata
```

I am not sure if a trailing slash is compulsory when it is a directory but I do it anyway for my packages.
```suggestion
pip-wheel-metadata/
```